pandas/tests/groupby/aggregate/test_other.py

```diff
@@ -454,6 +457,8 @@ def test_agg_tzaware_non_datetime_result():
     result = gb["b"].agg(lambda x: x.iloc[-1] - x.iloc[0])
     expected = Series([pd.Timedelta(days=1), pd.Timedelta(days=1)], name="b")
     expected.index.name = "a"
+    if as_period:
+        expected = expected.astype(object)
```

Comment:
Is this the correct expected result? When subtracting Periods, you get offset objects, not Timedelta objects?

Reply:
good point.  probably not great that tm.assert_series_equal can't tell the difference either